### ISSUE TITLE
Bootstrap Input Addon Style Issue with MaskedTextBox

### ISSUE THREAD

**Issue by PaitoAnderson** [telerik/kendo-react#92]
## I'm submitting a...
* Bug report

## Current behavior
<img width="361" alt="screen shot 2018-10-03 at 10 56 54 am" src="https://user-images.githubusercontent.com/4108718/46419146-47e3c300-c6fb-11e8-83a3-0d2ce47fbac9.png">

## Expected behavior
Shouldn't have a border radius on the left side, and it should have a rounded border radius on the right side.

## Minimal reproduction of the problem with instructions
https://stackblitz.com/edit/react-jhagez?file=app%2Fmain.js

## What is the motivation or use case for changing the behavior?
Border radius used doesn't look great

## Environment

Browser:
- Chrome (desktop) version 69.0.3497.100 

System:
- TypeScript version: 3.1.1
- Node version: 10.9.0 NPM Version: 6.4.1
- Platform:  macOS

**Comment by simonssspirit** [comment]
@PaitoAnderson This occurs because we are automatically changing the border-radius when class "input-group" is used. Currently, we covered only the case where the "input-group-text" element is on the right side of the MaskedTextVBox. I have logged an issue to fix that.

Currently, the following CSS can be used as a workaround:

```
.input-group .k-maskedtextbox .k-textbox,
.input-group .k-maskedtextbox .k-textbox .k-select {
 border-top-left-radius: 0;
 border-bottom-left-radius: 0;
 border-top-right-radius: 0.25em;
 border-bottom-right-radius: 0.25em;
}
```

This is the updated example: https://stackblitz.com/edit/react-jhagez-2ffnvj?file=index.html

This is the issue in the theme repository: https://github.com/telerik/kendo-themes/issues/358
